GUI/src/pages/CreateInvitationPage.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button, FormInput, Track } from '../components';
+import { createInvitation } from '../resources/api-constants';
+
+const CreateInvitationPage: React.FC = () => {
+  const { t } = useTranslation();
+  const [invitationResult, setInvResult] = useState('');
+  const [invitationResultLink, setInvResultLink] = useState('');
+  const [invitationError, setInvError] = useState('');
+  const [email, setEmail] = useState<string>('');
```

Comment:
Please remove `<string>` if not needed to make it consistent with other `useState` statements.